src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -63,7 +63,6 @@ public IVoidConfiguration WhenArgumentsMatch(Func<ArgumentCollection, bool> argu
 
         public virtual IAfterCallSpecifiedConfiguration DoesNothing()
         {
-            this.RuleBeingBuilt.Applicator = x => { };

```

Comment:
You're not talking nonsense at all. It is a little worrisome, isn't it?

First, I gotta say that the new implementation is a lot closer to "does nothing" than before. If only I could get rid of the pesky `return`, but we need something to hang the `Repeated` off.

Okay. I'll be serious now.

Your concerns are completely reasonable. 
The `Applicator` is... the thing that does stuff. Most of the time, it's used to set a return value. Otherwise, it's the thing that `Throws` the exception.
I think that's about it, really.

And every new rule is created with an applicator that returns the default value (null or value instance), so I think that aspect of the behaviour is correct.

Although you've caused me to realize something. A while back @thomaslevesque fixed #534 to make sure we couldn't save away a rule and then add multiple behaviours to it. The change as-is will break that for `DoesNothing`. Even though there's no "normal" way to make this happen using the fluent API, my change can't stand. I'll add an extra spec and amend the fix.

Thanks!